lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -298,13 +302,20 @@ defmodule Kernel.SpecialForms do
   Size and unit can also be specified using a syntax shortcut
   when passing integer values:
 
-      << x :: 8 >> == << x :: size(8) >>
-      << x :: 8 * 4 >> == << x :: size(8)-unit(4) >>
-      << x :: _ * 4 >> == << x :: unit(4) >>
+      iex> x = 1
+      iex> << x :: 8 >> == << x :: size(8) >>
+      true
+      iex> << x :: 8 * 4 >> == << x :: size(8)-unit(4) >>
+      true
 
   This syntax reflects the fact the effective size is given by
   multiplying the size by the unit.
 
+  However, if you specify the unit without specifying the size, you'll get
```

Comment:
I believe we can remove this bit. There are a bunch of things that will raise a compilation error. We can just skip it. :)